pkg/ddevapp/.env

```diff
@@ -0,0 +1,23 @@
+# The application ID used to to uniquely store session and cache data, mutex locks, and more
```

Comment:
I'm quite sure this file is in the wrong place, please remove it or put it where it belongs.

Reply:
Ah yes, that got checked in somehow. Removed.